reqpart.ks.in

```diff
@@ -25,21 +25,17 @@ shutdown
 
 %post
 
-expected_fstype="ext4"
-if [ "@KSTEST_OS_NAME@" == "rhel" ]; then
-    expected_fstype="xfs"
-fi
-
 # FIXME: This will need to test for other things one day when we support tests on other arches
 boot_mount="$(grep boot /proc/mounts)"
 if [ -z "${boot_mount}" ]; then
     echo "*** no /boot mount point was created by reqpart" >> /root/RESULT
 fi
 
 # Also verify the partitions we explicitly asked for got made.
+# Fedora Server, Workstation, and RHEL use different fs types by default, so accept any of them
 root_fstype=$(blkid -o value -t LABEL=root -s TYPE -l)
-if [ "${root_fstype}" != "${expected_fstype}" ]; then
-    echo "default fstype is incorrect (got $root_fstype; expected ${expected_fstype})" >> /root/RESULT
+if ! echo "$root_fstype" | grep -Eq 'xfs|ext4|btrfs'; then
```

Comment:
This doesn't really test anything. We need to make sure that the type matches the expected type.

Reply:
OK -- I thought this test covers that partitions get created (with some default file system), it otherwise duplicates the test of default-fstype-*.

Would it be ok to install with `inst.variant=Server` and expect "xfs" here?